src/components/WwwFrame/DisclaimersContentful.vue

```diff
@@ -1,14 +1,14 @@
 <template>
-	<div class="row">
-		<ol id="disclaimers" class="text-left">
+	<kv-page-container>
```

Comment:
I like that you removed `<div class="row">` here. However, I would suggest using wrapping it with `<kv-page-container>` in the context it's used in, which right now is TheFooter.vue. Rather than in the component itself.

This will make it so `<disclaimers-contentful>` can be used in other contexts and layouts, maybe taking up only half-of the screen width, etc. 

Just generally keeping layout handled by what is using the component, rather than putting layout in the component itself.

Reply:
Understood and agree!